# synergus-genomics
Data and scripts for phylogenomic analysis of Synergus itoensis and relatives

Uppload of data in progress. Available so far:
Fasta files of all gene sequences of single copy orthologs (for each group, one fasta file containing 4 unaligned sequences)
Fasta files of all protein sequences of single copy orthologs (for each group, one fasta file containing 4 unaligned sequences. Additionally, 4 fasta files with all the single copy orthologs from each of the species)
Fasta files of all protein alignents of single copy orthologs generated using Muscle (for each group, one fasta file containing 4 aligned sequences). Additionally, one file listing the lowest identity and lowest similarity to the consensus sequence among the four aligned sequences.
Fasta files of all protein alignents of single copy orthologs generated using ClustalOmega (for each group, one fasta file containing 4 aligned sequences) Additionally, one file listing the lowest identity and lowest similarity to the consensus sequence among the four aligned sequences.
Clustal files of the conserved blocks according to Gblocks
Alignments of the coding sequences generated with Pal2Nal, in paml format (modified phylip)
Alignments of the coding sequences generated with Pal2Nal, in fasta format
Alignments of the coding sequences generated with Pal2Nal, in phylip format (each directory also contains the species tree and the list of alignment files, both required by Bayescode)
The 300 aligned proteins randomly chosen for making the tree with MrBayes
All the output files generated by running Orthofinder on the four species.
The list of syngle copy orthogroups generated by Orthofinder
Tables listing all the results of Codeml analysis (in csv format)
